source/Mlos.Agent.Server/MlosAgentServer.cs

```diff
@@ -220,6 +232,71 @@ public static void Main(string[] args)
             Console.WriteLine("Mlos.Agent exited.");
         }
 
+        private static void ShowUsageHelp<T>(ParserResult<T> parserResult, IEnumerable<Error> errors = null, string msg = null)
+        {
+            if (msg != null)
+            {
+                Console.Error.WriteLine(msg);
+                Console.Error.WriteLine(string.Empty);

```

Comment:
Nit (feel free to ignore): This can be just WriteLine() without any args.

Reply:
Fixed.